src/components/Tombstone.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <dl id="tombstone">
+    <!-- Registration Date -->
+    <template v-if="!!registrationDate">
+      <dt class="mr-2">Registration Date:</dt>
+      <dd id="registration-date">{{ registrationDate }}</dd>
+    </template>
```

Comment:
This looks like a file that's part of a separate pull request.  Is it possible that changes from two PRs are shown here?

Reply:
You're correct. This development builds upon the allowed actions work and crosses with the entity info refactor I did recently. I haven't untangled this PR yet. I'm hoping the rebase is relatively smooth....

Reply:
![image](https://user-images.githubusercontent.com/25233684/228637167-2be1bd3e-9fe2-4d6c-bf4d-efc7371e1057.png)

Reply:
I didn't want to overwrite any files in this PR so I created another PR with the same commits, but based on release-5.12.0 branch. Let me know if you like it better:

https://github.com/bcgov/business-filings-ui/pull/477